src/components/Header/LandingPage/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import styles from './LoginHeader.module.scss';
+import Logo from '/public/svgs/logo.svg';
+import Taskify from '/public/svgs/taskify.svg';
+
+function LoginHeader() {
```

Comment:
```suggestion
function LoginHeader() {
```
요 파일은 메인 페이지 헤더인 것 같은데 제가 메인 페이지 만들 때 만들어서 없어도 괜찮을 것 같네요?!